src/main/java/org/folio/qm/validation/ValidationField.java

```diff
@@ -0,0 +1,6 @@
+package org.folio.qm.validation;
+
+import org.folio.rspec.validation.validator.marc.model.MarcRuleCode;
+
+public record ValidationField(String tagName, boolean isSkipped, MarcRuleCode ruleCode) {
```

Comment:
is there a purpose/case to have ```isSkipped=false```?, if the idea of this record to be used only for filtering/skipping then we may rename it appropriately and remove ```isSkipped``` property

Reply:
The `isSkipped=false` flag is used in the `updateById` method, but it is probably better in this case to just pass an empty collection.